sycl/doc/GetStartedGuide.md

```diff
@@ -782,9 +782,7 @@ queue objects bound to Intel GPU device:
 int main() {
 
   auto NEOGPUDeviceSelector = [](const sycl::device &Device){
-    using namespace sycl::info;
-
-    const std::string DeviceName = Device.get_info<device::name>();
+    const std::string DeviceName = Device.get_info<sycl::info::device::name>();
```

Comment:
What is wrong with the version on the left?

Reply:
I think it is more concise.